src/full/cache.clj

```diff
@@ -46,7 +46,7 @@
     ::none))
 
 (defn- do-get-or-load>
-  [k loader> timeout {:keys [setf getf states]}]
+  [k loader> timeout {:keys [setf getf states allow-nil?]}]
```

Comment:
A default of `true` needs to be added here or in `lget-or-load>`, otherwise `lget-or-load>` changes functionality.